src/data/lists.ts

```diff
@@ -1,19 +1,56 @@
 import config from "config";
 import sql from "mssql";
+import { ListItem, ListItemQuery } from "../routes/api/lists";
 
 const sqlInfo: sql.config = config.get("SQL_INFO");
 const pool = new sql.ConnectionPool(sqlInfo);
 const connectionPromise = pool.connect();
 
-export const getListFromDB = async (id: string): Promise<boolean> => {
+export const getListFromDB = async (id: string): Promise<ListItem[]> => {
     const connection = await connectionPromise;
     const ps = new sql.PreparedStatement(connection);
-    ps.input("id", sql.UniqueIdentifier);
+    console.log(id);
+    ps.input("id", sql.VarChar);
+    let listItems: ListItem[] = [];
     try {
-        await ps.prepare(`SELECT gameId FROM ListItems LI
-            INNER JOIN List L ON L.id = LI.listId
-            WHERE userId = (@id)`);
+        await ps.prepare(`SELECT gameId, played FROM ListItems LI
+            INNER JOIN Lists L ON L.id = LI.listId
+            WHERE userId = @id`);
         let result = await ps.execute({ id });
+        listItems = [];
+        console.log("result.recordset");

```

Comment:
I should trim out some pointless logs I guess

Reply:
done it for you